pkg/services/featuremgmt/registry.go

```diff
@@ -1217,6 +1217,14 @@ var (
 			HideFromDocs:      true,
 			HideFromAdminPage: true,
 		},
+		{
+			Name:              "ssoSettingsSAML",
+			Description:       "Enables the SSO settings API and the SAML configuration UIs in Grafana",
```

Comment:
This is a bit misleading, because this feature toggle will be used only by the SAML connector, I'd prefer something like `Use the new SSO Settings API to configure the SAML connector`

Reply:
Changed as suggested.